scripts/npm-safe-publish.sh

```diff
@@ -104,23 +104,34 @@ fi
 if [[ "$DEPRECATED_PACKAGE" != "" ]]; then
   echo "Deploy to Deprecated Package: $DEPRECATED_PACKAGE"
 
-  # Backup the original package.json
-  cp package.json package.json.bak
+  # Check version for DEPRECATED_PACKAGE
+  NPM_SHOW_DEPRECATED="$(npm show "$DEPRECATED_PACKAGE" version)"
```

Comment:
Doesn't this only show the latest version? `npm ls` is the one I've used to show the package registry version.
Or do I have that backwards?